automation/src/test/scala/org/broadinstitute/dsde/test/api/MethodConfigApiSpec.scala

```diff
@@ -0,0 +1,134 @@
+package org.broadinstitute.dsde.test.api
+
+import com.typesafe.scalalogging.LazyLogging
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+import org.broadinstitute.dsde.workbench.config.UserPool
+import org.broadinstitute.dsde.workbench.fixture._
+import org.broadinstitute.dsde.workbench.service.{Orchestration, Rawls}
+import org.broadinstitute.dsde.workbench.service.test.RandomUtil
+import org.scalatest.{FreeSpec, Matchers}
+import spray.json.pimpString
```

Comment:
Not sure what version of `spray-json` we're using here, but in googling for `pimpString` I found that it's been deprecated in favor of enrichString https://github.com/spray/spray-json/blob/0b893f26ed2fd71649389e7eccc7bba6f1507946/src/main/scala/spray/json/package.scala#L35

Reply:
@marctalbott Hi, instead `val jsonStr = response.parseJson.asJsObject.getFields("methodRepoMethod")`, I'm using 
```
val json: JsValue = JsonParser(response)
val field: JsValue = json.asJsObject.fields("methodRepoMethod")
```
avoiding `spray.json.pimpString`. What do you think?